.github/workflows/fix_compile_commands.py

```diff
@@ -21,6 +21,8 @@ def main():
         # Filter out -isystem flags that cause false positives
         iter_args = iter(obj["arguments"])
         for arg in iter_args:
+            if arg == "-Wno-maybe-uninitialized":
+                out_arg += "-Wno-uninitialized"
```

Comment:
This need to be part of the if-else structure, or the original warning flag will get added twice: once here, and once in the else clause below.